src/main/java/src/chapter5/question1/Question1.java

```diff
@@ -0,0 +1,14 @@
+package src.chapter5.question1;
```

Comment:
The questions asks you to test both toString and substring methods. I believe toString version is missing.

Reply:
toString versions have been added.